src/Components/Onboarding/Steps/Artists/ItemLink.tsx

```diff
@@ -38,28 +39,71 @@ const Col = styled.div`
   align-items: center;
 `
 
-interface State {
+export interface State {
   fadeIn: boolean
   fadeOut: boolean
+  artist_id: string
 }
 
-type Props = React.HTMLProps<HTMLAnchorElement> & RelayProps
+interface Props extends React.HTMLProps<HTMLAnchorElement>, RelayProps {
+  relay?: RelayProp
+}
 
 class ItemLink extends React.Component<Props, State> {
   state = {
     fadeIn: false,
     fadeOut: false,
+    artist_id: this.props.artist.id,
+  }
+
+  followArtist() {
+    console.log("follow artist!")
+    const storeUpdater: SelectorStoreUpdater = (store, suggestedArtistData) => {
+      // Replace now followed artist in list with newly suggested artist
+    }
+
+    commitMutation(this.props.relay.environment, {
+      mutation: graphql`
+        mutation ItemLinkFollowArtistMutation($input: FollowArtistInput!) {
+          followArtist(input: $input) {
+            artist {
+              # Get related artist (not this field that’s currently used!)
+              artists(size: 1) {
+                name
+              }
+            }
+          }
+        }
+      `,
+      variables: {
+        input: {
+          artist_id: this.props.artist.id,
+          unfollow: false,
```

Comment:
Such negation… this param should really be called `follow` imo, but that’s probably a breaking schema change.